packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -66,10 +66,15 @@ export async function runWithRetry<T>(
             }
 
             if (progress.cancel?.aborted === true) {
+                logger.sendTelemetryEvent({
+                    eventName: `${fetchCallName}_runWithRetryAborted`,
+                    retry: numRetries,
+                    duration: performance.now() - startTime,
+                }, err);
                 throw new NonRetryableError(
-                    "runWithRetryAborted",
+                    "runWithRetry was Aborted",
                     OdspErrorType.fetchTimeout,
```

Comment:
I still don't understand why this errorType is used here, but if you and Vlad agreed on it I'm okay.  Do you have a follow-up issue opened to move this (or whatever the final correct errorType is) to base driver ErrorType and use it here?

Reply:
yeah #8488

Reply:
i can also change it to generic now and then use the final correct type later

Reply:
Hmmm I guess I would prefer that since then you could remove the dependency on ODSP driver packages from here.  And you can call out this case in that issue so we don't forget to update it too.  Honestly I don't feel too strongly so you can decide either way.